package.json

```diff
@@ -14,27 +14,37 @@
   "scripts": {
     "start": "stripes serve",
     "build": "stripes build --output ./output",
-    "test": "stripes test karma",
-    "test-coverage": "stripes test karma --coverage",
+    "test:unit": "jest --ci --coverage",
+    "test:e2e": "stripes test karma --karma.singleRun --karma.browsers ChromeDocker --karma.reporters mocha junit --coverage",
+    "test": "yarn run test:unit && yarn run test:e2e",
```

Comment:
I think with a new approach in order to have a proper coverage metric only the jest tests would be needed. As biggest also produces coverage information. And I think in this case two tools might collide. Overall I think as the result of this work PR should produce some coverage in the sonar report. Try to play around with just jest tests on CI.

Reply:
This is the thing I am thinking about. For now, the coverage report can be generated only for e2e tests or unit tests because of different reporters. However, I noticed that ui-receiving module has such kind of [script](https://github.com/folio-org/ui-receiving/blob/master/package.json#L160), so I decided to try it for ui-data-import. But yeah, report's still overwritten.

Reply:
I see that ui-receiving run unit tests after BigTest. So maybe this is also a thing to try out.

Reply:
Already tried. When running unit tests after BigTest then there is information only about unit tests coverage (MARCTable component). When running them before then there is information only about e2e tests coverage (MARCTable component coverage is about 0%). One report overwrites another.

Reply:
So I guess that it happens because of different reporters, but I still haven't figured out how to use the same reporter for both tests.

Reply:
I think it is great that jest produces information. I think BigTests is enough to just run really without bothering about coverage aspect. Because of the unstable tests, the coverage for it would always be zero if at least one test fails.

Reply:
@VictorSoroka1 FYI Ian created a ticket for investigating this- https://issues.folio.org/browse/FOLIO-2703